scripts/release/release-layout.yml

```diff
@@ -10,6 +10,10 @@ layout:
     - workflow-artifact:
         name: "Code Scanning Query Pack Generation"
         artifact: code-scanning-cpp-query-pack.zip
+  coding-standards-qlpacks.zip:
```

Comment:
I am not sure what the right answer currently is, but the name of this compared to the bundle which we are currently calling a pack might be confusing? I dont think the right answer is to change the old artifact's name though...

maybe what solves this problem is adding some documentation to [this part of the user manual](https://github.com/github/codeql-coding-standards/blob/main/docs/user_manual.md#release-information) that adds the explanation of what these are vs what is currently there

Reply:
I've renamed to "coding-standards-codeql-packs", and will update the user manual to clarify the difference.